hw/misc/hermes.c

```diff
@@ -30,6 +30,7 @@
 #include "qapi/error.h"
 #include "trace.h"
 #include <ubpf.h>
+#include <elf.h>
```

Comment:
What's MVP?

Reply:
_Minimum viable product_
I use the term to state that we are at the stage, where the product (in this case PR) are around the point where we could potentially integrate this, altough it would be far from perfect.